### ISSUE TITLE
[Eligibility Questionnaire] Add filtering functionality for senior housing

### ISSUE THREAD

**Issue by elenm** [CityOfDetroit/bloom#216]
- [x] Decide how to store state and implement 
- [x] Determine whether we have the necessary fields in the backend to filter on the question answers
- [x] Get senior housing filtering working

**Comment by elenm** [comment]
Finishing this is also blocked on getting information on how we should compute eligibility. I can start working on certain aspects of it.

**Comment by willrlin** [comment]
Can we downscope this ticket to what we can finish this sprint and open another ticket for any blocked work?
